### PR TITLE
IHub-214835 Creating README

### PR DESCRIPTION
**Pull request by P4radise**
[IHub-214835](https://trackor.onevizion.com/form/ConfigAppForm.do?id=100105348559&ttid=10009161) Creating README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
 # AppHealthReport
 
-Module for installing AppHealthReport components
\ No newline at end of file
+The module installs AppHealthReport components:
```
Comment (glebowadim) [inline README.md]: The Module doesn't install components. The components are part of the AppHealthReport Module

File: README.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+1) "Weekly App Health Report» report - the report collects error information for Reports, Imports, Modules, Rules, Notifications, and Trackor Mail, as well as their run times.
```
Comment (glebowadim) [inline README.md]: I think the structure should be like this:

- What is it?
- What does it consist of?
- What does it do and what additional steps and settings are needed?

**Review by glebowadim — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+By default, the data is collected for the week, but the data sampling can be customized.
+
+2) "Weekly App Health - run report" rule - the rule that runs the report.
+For this report to run successfully, the AsyncRuleDefaultUser Program Parameter must be filled.
+
+3) Rule "Weekly App Health - send a mail" - a rule that triggers after successful report execution and sends it to the specified email.
+By default, it's noc-reports@onevizion.com, the mail gets there and then a Website Error Report is created on trackor.onevizion.com.
```
Comment (asmoiseenko) [inline README.md]: Remove references to OneVizion resources from the public repo

**Review by asmoiseenko — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (P4radise) [reply README.md]: I updated the README and made a similar structure in the form of a table:
| Component | Description | Additional Information |

File: README.md (hunk 1 above)
Comment (P4radise) [reply README.md]: Okay, I got it, I fixed it

File: README.md (hunk 3 above)
Comment (P4radise) [reply README.md]: Done

**Review by asmoiseenko — CHANGES_REQUESTED** [review]
What is the purpose of the report? What is checked by this report? What happens if report fails?

**Comment by P4radise** [comment]
> What is the purpose of the report? What is checked by this report?

I have added information before the table that answers this question:
```
The Module collects data on errors with components in the installation and sends them to a special mail for studying these errors.
```
Almost the same information is in the table, it is just in different parts of it. In general, this report is needed to monitor components that crash with errors

> What happens if report fails?

If the report ends with an error, nothing happens. The report will not come to the mail and for it will not be created a Trackor in special Trackor Type, in this case, a Case will be created and I will look into the cause.

**Review by asmoiseenko — CHANGES_REQUESTED** [review]
Write something like this:
This module includes a report which collects data about failures of the any of the following components for the XXX period. 
list of components

When report is complete successfully it is emailed to XXX address (is it configurable?). The module uses rule to email report because XXX

**Comment by P4radise** [comment]
> Write something like this: This module includes a report which collects data about failures of the any of the following components for the XXX period. list of components
> 
> When report is complete successfully it is emailed to XXX address (is it configurable?). The module uses rule to email report because XXX

Done
Yes, we can configure the email in the rule

File: README.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
-# AppHealthReport
+# app-health-report
 
-Module for installing AppHealthReport components
\ No newline at end of file
+This Module includes a report which collects data about failures of the any of the following components for the XXX period.
```
Comment (asmoiseenko) [inline README.md]: ... for the **selected** period.

File: README.md [hunk 5]
```diff
@@ -1,3 +1,7 @@
-# AppHealthReport
+# app-health-report
 
-Module for installing AppHealthReport components
\ No newline at end of file
+This Module includes a report which collects data about failures of the any of the following components for the XXX period. 
+
+List of components: Reports, Imports, Modules, Rules, Notifications, Trackor Mail, Run Time.
+
+When report is complete successfully it is emailed to XXX address.
```
Comment (asmoiseenko) [inline README.md]: ... to the address configured in the rule.

**Review by asmoiseenko — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (P4radise) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (P4radise) [reply README.md]: Done

**Review by asmoiseenko — APPROVED** [review]

**Review by asmoiseenko — APPROVED** [review]

**Review by glebowadim — APPROVED** [review]

**Pull request merged**